components/Shared/Layout/SidebarLayout.jsx

```diff
@@ -1,40 +1,24 @@
-import React from 'react'
 import styled from 'styled-components'
 
 // Sidebar layout w/ implicit sizing & wrap, courtesy of https://every-layout.dev/layouts/sidebar/
 
 // Wrapper wraps the content and applies a negative margin onto "Gutter" - thus acting as a defacto gutter between the Sidebar and Content sections.
 export const Wrapper = styled.div`
-    display: flex;
-    flex-wrap: wrap;
-    flex-grow: 999;
-
-    > * {
-      display: flex;
-      flex-wrap: wrap;
-      flex-grow: 999;
-      margin: -0.5rem;
-    }
-
-    > * > * {
-      /* ↓ applies to both elements */
-      margin: 0.5rem;
-    }
-  }
-  `
-// Creates an implicit gutter between Sidebar and Content
-export const Gutter = styled.div``
+  display: flex;
+  flex-wrap: wrap;
+`
 
 // Sidebar grows to adopt the width of its children
 export const Sidebar = styled.div`
   flex-grow: 1;
+  margin: 0.5rem;
 `
 // Content is a flexible container with no explicit width (hence basis=0) but which grows to consume all available space. It then wraps once its min-width is reached.
 export const Content = styled.div`
   display: flex;
+  justify-content: center;
   flex-basis: 0;
   flex-grow: 999;
-  justify-content: center;
   padding-top: ${props => props.theme.sizes[4]}px;
-  min-width: calc(55% - 1rem);
```

Comment:
@Schwartz10 Just wanna point out that removing this means the `SendCard` gets squished a lot more before wrapping. Restoring it doesn't add any problems but allows us to control when, exactly, it wraps. 

Alternatively, to give us more granular control (i.e. to set a different `min-width` for the `SendCard` vs. `TxHistory`, we can institute this in a parent element inside of `Content`.

Reply:
I'd say put it back in if you don't see any downside (just make sure no min-width is set somewhere else). I'd say less parent / child elements the better